common/djangoapps/pipeline_mako/templates/static_content.html

```diff
@@ -65,23 +75,23 @@
 from django.template.loaders.filesystem import Loader
 engine = Engine(dirs=settings.DEFAULT_TEMPLATE_ENGINE['DIRS'])
 source, template_path = Loader(engine).load_template_source(path)
-%>${source}</%def>
+%>${source | n, decode.utf8}</%def>
 
 <%def name="require_module(module_name, class_name)">
     <script type="text/javascript">
         (function (require) {
             % if settings.REQUIRE_DEBUG:
-                require(['${module_name}'], function (${class_name}) {
-                    ${caller.body()}
+                require(['${module_name | n, js_escaped_string}'], function (${class_name | n, js_escaped_string}) {
```

Comment:
@nedbat: I'm not sure what is the right thing to do here.  

The second expression is a class name which presumably does not contain user data.  Using `| n, js_escaped_string` as I have done would correctly fail the parser because it is not surrounded by quotes, and thus is not considered safe.  Using `dump_js_escaped_json` would add quotes, which we don't want.

My options include:
1. Leave as `js_escaped_string`, which is misleading and wrong. :)
2. Switch to `| n, decode.utf8`, which is our way of stating that something is simply already safe, or
3. We could introduce a filter named `js_identifier` or `js_simple_identifier` which would use a simple regex like what I use a simple rule like (`"[_$a-zA-Z]+[_$a-zA-Z0-9]*"`), and could replace anything outside of this with an '_'.

Note that the actual rules for a [JavaScript identifier](https://mathiasbynens.be/notes/javascript-identifiers) are far more complicated, and I don't think we should bother.